src/Raven.Server/Documents/Handlers/IndexHandler.cs

```diff
@@ -626,10 +626,7 @@ public async Task Delete()
 
             if (LoggingSource.AuditLog.IsInfoEnabled)
             {
-                var clientCert = GetCurrentCertificate();
-
-                var auditLog = LoggingSource.AuditLog.GetLogger(Database.Name, "Audit");
-                auditLog.Info($"Index {name} DELETE by {clientCert?.Subject} {clientCert?.Thumbprint}");
+                LogAuditFor(Database.Name, $"Index {name} DELETE");
```

Comment:
Should we also audit resets @arekpalinski ?

Reply:
Yes, let me handle that as part of https://issues.hibernatingrhinos.com/issue/RavenDB-22267/Audit-log-on-indexes-additional-sources-custom-sorters-and-analyzers